cypress/e2e/0-api/channel_groups_api.cy.js

```diff
@@ -90,10 +100,10 @@ describe('channel groups API', () => {
   context('POST /channelGroups/:id/output', () => {
     const OUTPUT = 0;
     it('toggles output for all channels in group', () => {
-      cy.toggleGroupOutput(CHANNEL_GROUP_ID, OUTPUT).then(response => {
+      cy.toggleGroupOutput(channelGroupId, OUTPUT).then(response => {
```

Comment:
```suggestion
      cy.toggleGroupOutput(channelGroupId, 0).then(response => {
```
since you're using `1` on line 106 anyway. and remove line 101